vexim/siteaddsubmit.php

```diff
@@ -132,10 +132,6 @@
       } else {
         header ("Location: site.php?added={$_POST['domain']}" .
                 "&type={$_POST['type']}");
-        mail("{$_POST['localpart']}@{$_POST['domain']}",
```

Comment:
Because this is a relay domain. For relay domains (neither an alias -> see condition line 83, nor local see condition line 104) we don't know a valid mail address (the principal error was that `$POST[localpart]` is not defined. We could only send a mail to `postmaster@relaydomain` supposing that this address always exists.